docs/docs/api/observe.md

```diff
@@ -6,7 +6,13 @@ sidebar_label: Observe
 
 `Observe` gives the ability to observe values from global state. To get a "live"
 version of a value from Engine's global state, state properties can be directly
```

Comment:
In essence, it's a one-way data binding. It might be important here to note that the values received through this binding are cloned and are not given by reference.